pymess/management/commands/sync_emails.py

```diff
@@ -9,19 +9,24 @@
 
 class Command(BaseCommand):
     """
-    Command synchronize e-mails body from directory.
-    Every e-mail body is named with e-mail slug and stored as a HTML file.
+    Command to synchronize e-mail bodies from directory.
+    Default templates remain the same. Variant B templates have the '_B' suffix.
     """
 
-    def _sync_for_locale(self, base_directory, locale):
+    def _sync_for_locale(self, base_directory, locale, variant):
         locale_directory = os.path.join(base_directory, locale) if locale else base_directory
 
         files_with_email_html_body = [
             filename for filename in os.listdir(locale_directory) if filename.endswith('.html')
         ]
         for filename in files_with_email_html_body:
+            if variant:
+                slug = filename[:-5]
+            else:
+                slug, variant = filename.rsplit('_', 1) # We extract slug
+                variant = variant[:-5] # we extract variant, format being slug_variant.html
```

Comment:
I still insist on something like:

```python
slug, _* = filename.split(f'_{variant}' if variant else '.html')
```

You don't need to assign a variant as the variant is in the command option. The statement I suggest works when the variant is set but also when it is not.